docs/reference/rocAL-and-RNNT.rst

```diff
@@ -0,0 +1,58 @@
+.. meta::
+  :description: rocAL RNNT dataloading reference
+  :keywords: rocAL, ROCm, API, RNNT, dataloading, python, pytorch, speech recognition
+
+*********************************
+rocAL RNNT dataloading in Python
+*********************************
+
+rocAL supports the RNNT speech recognition model through audio readers and other functions that can be used with PyTorch.
+ 
+All the functions used for RNNT dataloading are available in the ``amd.rocal.fn`` module. See `Using rocAL with Python API <../how-to/using-with-python.html>`_ for more details about this module. 

```

Comment:
First link is broken. Please fix

Reply:
The link works for me (in ReadTheDocs). But I changed its syntax just in case.